app/mailers/notifier.rb

```diff
@@ -39,6 +39,13 @@ def review_orders(user:, accounts:, facility: Facility.cross_facility)
     @user = user
     @accounts = accounts
     @facility = facility
+
+    @contact_message = if facility.email
+                         "please email #{facility.email}"
```

Comment:
Might try this for the MD link:
```suggestion
                         "please email [#{facility.email}](mailto:#{facility.email})"
```